clar2wasm/tests/wasm-generation/blockinfo.rs

```diff
@@ -0,0 +1,62 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use proptest::proptest;
+
+use crate::{block_range, uint};
+
+const BLOCK_INFO: [&str; 8] = [
+    "burnchain-header-hash",
+    "id-header-hash",
+    "header-hash",
+    "miner-address",
+    "block-reward",
+    "miner-spend-total",
+    "miner-spend-winner",
+    "time",
+];
+const BURN_BLOCK_INFO: [&str; 2] = ["header-hash", "pox-addrs"];
+const STACKS_BLOCK_HEIGHT_LIMIT: u32 = 136900;
+const BURN_BLOCK_HEIGHT_LIMIT: u32 = 827000;
```

Comment:
Most block heights up to these values will always be non-existent in the test environment, so those calls will always return `none`. Is that the intention?

Reply:
Fixed. Advancing both chain tips accordingly.

Reply:
Thanks. Did you verify that with the new change in place we are seeing results that are not just `none`?

Reply:
Changed that again. I've double-checked and the results were `none`.

Reply:
Ok, I see that some of the results are `none` and some are not. I think that's fine for now as it's clear that it's caused by the simulation.